### DISCUSSION TITLE
Confim: Treatment for Exercised Options for Data Entry

### DISCUSSION THREAD

**Discussion by angad-kapoor** [hickeng/financial#156] [Q&A] [answered]
Thank you for putting the spreadsheet together! Its been extremely helpful. 
I wanted to confirm - for Options exercised - I am assuming one would simply treat them as RSUs in terms of data entry? 
I.e. enter the no. of shares purchased with the date of exercise?

**Comment by hickeng** [comment, marked as answer]
Correct - make sure that the exercise price is correct. If they were exercised automatically then the default value is likely correct. If you exercised them at a specific time of day then you'll likely have to override the market value.
